doc/api/os.md

```diff
@@ -1208,6 +1242,57 @@ information.
   </tr>
 </table>
 
+### Priority Constants
+<!-- YAML
+added: REPLACEME
+-->
+
+The following process scheduling constants are exported by
+`os.constants.priority`:
+
+<table>
+  <tr>
+    <th>Constant</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+    <td><code>PRIORITY_LOW</code></td>
+    <td>The lowest process scheduling priority. This corresponds to
+    `IDLE_PRIORITY_CLASS` on Windows, and a nice value of `19` on all other
```

Comment:
If I am not mistaken. markdown is not processed inside HTML tags, so we need `<code>` elements instead of backticks in this table.